src/imgix-core-js.js

```diff
@@ -176,11 +176,12 @@
     ImgixClient.prototype._buildDPRSrcSet = function(path, params) {
         var srcset = '';
         var targetRatios = [1, 2, 3, 4, 5];
-        var url = this.buildURL(path, params);
         
         for(var i = 0; i < targetRatios.length; i++) {
           currentRatio = targetRatios[i];
-          srcset += url + ' ' + currentRatio +'x,\n'
+          currentParams = params;
```

Comment:
Couple of problems here:

- `currentParams` isn't declared as a variable, so it's going to be applied to the global scope instead of being scoped to this method.
- It's probably not even worth re-declaring this. We should just use `params`.
- Except we also need to make sure this doesn't break if no `params` are passed in. Something like `params = params || {};`